test/examples/py2pack-opensuse.dsc

```diff
@@ -1,8 +1,8 @@
 Format: 1.0
 Source: py2pack
-Version: 0.8.0
+Version: 0.8.5
 Binary: python-py2pack
-Maintainer: __USER__
+Maintainer: Benjamin Greiner
```

Comment:
That's breaking CI

Reply:
https://github.com/openSUSE/py2pack/pull/137 is the fix.

Reply:
Sorry, I overlooked this replacement. Obviously it worked when I tested locally. Thanks for the fix.

Reply:
Thanks for working on all this!